reference/test/solver/cg_kernels.cpp

```diff
@@ -105,7 +105,7 @@ TEST_F(Cg, SolvesStencilSystem)
     GKO_ASSERT_MTX_NEAR(x, l({1.0, 3.0, 2.0}), 1e-14);
 }
 
-
+/*
 TEST_F(Cg, SolvesMultipleStencilSystems)
```

Comment:
Why did you comment out this test?
Commented out code should be removed from the repo and not kept (if the removal was intentional).